messaging/WFLY-13991_keep_corrupted_journal_files.adoc

```diff
@@ -0,0 +1,66 @@
+= Keep corrupted Journal filesin Artemis.
+:author:            Emmanuel Hugonnet
+:email:             your.email@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:          messaging,jms
+:idseparator:       -
+
+== Overview
+
+
+When a corrupted artemis journal file is seen Artemis shuts down EAP, but before it does that it deletes the corrupted file. This means that no after the event analysis/investigation can be done as the corrupted journal file is gone.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-13991[WFLY-13991]
+
+=== Related Issues
+
+* https://issues.apache.org/jira/browse/ARTEMIS-2939[ARTEMIS-2939]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+=== Testing By
+
+* [X] Engineering
+
+* [ ] QE
+
+=== Affected Projects or Components
+
+* WildFly
+* ActiveMQ Artemis (finished)
+
+=== Other Interested Projects
+
+== Requirements
+
+* The journalMaxAtticFiles parameter of ActiveMQ Artemis configuration must be configurable via Wildfly management model.
```

Comment:
Can you please describe requirements for this parameter? Ok, it is INT, default value is 10. I also read something in documentation and implementation PR, but implementation is prepared based on the requirements from this document. Btw, what is attic folder?

Reply:
A folder called attic inside the journal folder. This is Artemis we are talking about